result.html

```diff
@@ -11,14 +11,25 @@
 </head>
 <body>
 
-    <header class="ques-header padding-xxlg flex-column flex-center margin-btm-lg">
+    <header class="page-header flex-horizontal-start padding-xxlg">
+        <p class=" c-blue text-bold font-size-xlg text-uppercase letter-space-md">
+            sneakerhead
+        </p>
+    </header>
+    <!-- <header class="ques-header padding-xxlg flex-column flex-center margin-btm-lg">
```

Comment:
If this code not necessary you can delete it

Reply:
thanks for pointing out @MayankKumar10 . i have made the changes.